test/integration/aab_offline_test.go

```diff
@@ -29,20 +29,24 @@ import (
 // TestOffline makes sure minikube works without internet, once it the user has already cached the images, This test has to run after TestDownloadOnly!
 func TestOffline(t *testing.T) {
 	t.Run("group", func(t *testing.T) {
-		for _, runtime := range []string{"docker", "crio", "containerd"} {
-			runtime := runtime
-			t.Run(runtime, func(t *testing.T) {
+		for _, rt := range []string{"docker", "crio", "containerd"} {
+			rt := rt
+			t.Run(rt, func(t *testing.T) {
 				MaybeParallel(t)
 
-				if runtime != "docker" && NoneDriver() {
+				if rt != "docker" && Arm64Platform() {
+					t.Skipf("skipping %s - only docker runtime supported on arm64", t.Name())
```

Comment:
if it passes the test, lets not skip it

Reply:
I think this is doing the right thing: skip arm64 tests if we aren't on Docker.